src/main/controllers/MediaAccountReviewController.ts

```diff
@@ -33,7 +33,7 @@ export default class MediaAccountReviewController {
       const imageName = applicant.imageName;
       const extension = imageName.substring(imageName.lastIndexOf('.') + 1, imageName.length);
 
-      const contentType = allowedImageTypeMappings[extension];
+      const contentType = allowedImageTypeMappings[extension.toLowerCase()];
```

Comment:
Could we add a small unit test just checking this? Will act as a regression test for the future

Reply:
Added tests :)